x-pack/plugins/transform/server/routes/api/error_utils.ts

```diff
@@ -138,3 +146,28 @@ export function wrapEsError(err: any, statusCodeToMessageMap: Record<string, any
   const message = statusCodeToMessageMap[statusCode];
   return new Boom(message, { statusCode });
 }
+
+interface EsError {
+  type: string;
+  reason: string;
+  line?: number;
+  col?: number;
+}
+
+/**
+ * Returns an error message based on the root cause
+ */
+function extractErrorMessage({ type, reason, line, col }: EsError): string {
+  let message = `[${type}] ${reason}`;
+
+  if (line !== undefined && col !== undefined) {
+    message +=
+      ', ' +
+      i18n.translate('xpack.transform.models.transformService.withColAndLineMessage', {
```

Comment:
we don't format error messages on the server side.
This will be appended onto an error which will be in english.

Reply:
Removed error translation in https://github.com/elastic/kibana/pull/84932/commits/307252b7778259f9162bb4b0a86594b5d98e5e18